packages/react/src/primitives/SwitchField/__tests__/SwitchField.test.tsx

```diff
@@ -147,80 +126,41 @@ describe('Switch Field', () => {
       expect(field).toHaveAttribute('value', 'checkboxValue');
     });
 
-    it('should disable the checkbox with the isDisabled prop', async () => {
-      const { container } = render(
-        <SwitchField label={label} isDisabled={true} />
-      );
+    it('should disable the checkbox with the isDisabled prop', () => {
+      const { container } = render(<SwitchField label={label} isDisabled />);
 
       const field = container.getElementsByTagName('input')[0];
       expect(field).toHaveProperty('disabled', true);
     });
 
-    it('should set the input to checked with the isChecked prop', async () => {
-      const { container } = render(<ControlledSwitch />);
+    it('should set the input to checked with the isChecked prop', () => {
+      const { container } = render(<SwitchField isChecked label={label} />);
 
       const field = container.getElementsByTagName('input')[0];
       expect(field).toBeChecked();
     });
 
     it('should update the checked value when a controlled value is updated', async () => {
-      render(<ControlledSwitch />);
+      const { rerender } = render(<SwitchField isChecked label={label} />);
 
-      let input = await screen.findByLabelText(label);
+      const input = await screen.findByLabelText(label);
       expect(input).toBeChecked();
 
-      act(() => updateControlledValue(false));
-      input = await screen.findByLabelText(label);
-      expect(input).not.toBeChecked();
-    });
-
-    it('should update the checked value with a click event', async () => {
-      render(<ControlledSwitch />);
+      rerender(<SwitchField isChecked={false} label={label} />);
 
-      let input = await screen.findByLabelText(label);
-      expect(input).toBeChecked();
-
-      userEvent.click(input);
-      input = await screen.findByLabelText(label);
       expect(input).not.toBeChecked();
     });
 
-    it('should set the input to checked with the defaultChecked prop', async () => {
+    it('should set the input to checked with the defaultChecked prop', () => {
       const { container } = render(
-        <SwitchField label={label} defaultChecked={true} />
+        <SwitchField label={label} defaultChecked />
       );
 
       const field = container.getElementsByTagName('input')[0];
       expect(field).toBeChecked();
     });
 
-    it('should fire the onChange function with a checkbox change event', async () => {
```

Comment:
Removed this test as it was really only testing the behavior of the `SwitchFieldControlledExample` contained inside, not the functionality of the `SwitchField`. To note, doublechecked that it had no impact on test coverage